zulipterminal/model.py

```diff
@@ -188,15 +199,8 @@ def _start_presence_updates(self) -> None:
         # FIXME: Version 2: call endpoint with ping_only=True only when
         #        needed, and rely on presence events to update
         while True:
-            response = self.client.update_presence(
-                request={
-                    # TODO: Determinal `status` from terminal tab focus.
-                    'status': 'active' if self.new_user_input else 'idle',
-                    'new_user_input': self.new_user_input,
-                }
-            )
-            self.new_user_input = False
             time.sleep(60)
+            self._notify_server_of_presence()
```

Comment:
If these two lines are reordered, do you need to call `_notify...` from `__init__`?